### PR TITLE
CRAN comments

### PR DESCRIPTION
**Pull request by niessl**
Fixes final CRAN comments

### CODE REVIEW

File: R/getWaitTimeSum.R [hunk 1]
```diff
@@ -42,10 +42,10 @@ getWaitTimeSum <- function(U, haz1, haz2, p1, p2, entry) {
       stime[i] <- res$root
       i <- i + 1
       if (res$root == 0) {
-        cat("Error uniroot: accuracy (tol=10^-16) is not enough.")
+        warning("Error uniroot: accuracy (tol=10^-16) is not enough.")
```
Comment (danielinteractive) [inline R/getWaitTimeSum.R:45]:
```suggestion
        warning("uniroot: accuracy (tol=10^-16) is not enough.")
```

File: R/getWaitTimeSum.R [hunk 2]
```diff
@@ -42,10 +42,10 @@ getWaitTimeSum <- function(U, haz1, haz2, p1, p2, entry) {
...
       }
     } else {
-      cat("Error uniroot: Values at interval endpoints (interval=c(0,10000)) are not of opposite signs. \n")
+      warning("Error uniroot: Values at interval endpoints (interval=c(0,10000)) are not of opposite signs. \n")
```
Comment (danielinteractive) [inline R/getWaitTimeSum.R:48]:
```suggestion
      warning("uniroot: Values at interval endpoints (interval=c(0,10000)) are not of opposite signs. \n")
```

**Review by danielinteractive — COMMENTED** [review]
thx Alex! can you also increment the version to 0.0.5 in description and NEWS files?

File: DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: simIDM
-Title: Simulating Oncocolgy Trials using an Illness-Death Model
-Version: 0.0.4.9000
+Title: Simulating Oncology Trials using an Illness-Death Model
+Version: 0.0.5.9000
```
Comment (danielinteractive) [inline DESCRIPTION:4]:
```suggestion
Version: 0.0.5
```

File: NEWS.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
-# simIDM 0.0.4.9000
+# simIDM 0.0.5.9000
```
Comment (danielinteractive) [inline NEWS.md:1]:
```suggestion
# simIDM 0.0.5
```

**Review by danielinteractive — APPROVED** [review]

**Review by numbersman77 — APPROVED** [review]

**Pull request merged**